src/selenium.ts

```diff
@@ -7,7 +7,7 @@
  * Code distributed by Google as part of the polymer project is also
  * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
  */
-import * as chalk from 'chalk';
+import chalk from 'chalk';
```

Comment:
`import chalk = require('chalk');`?

Reply:
sure

Reply:
Actually, this seems to break typing.

Reply:
Huh, chalk must have change to module-compatible exports